frontend/src/features/admin-form/responses/ResponsesPage/storage/UnlockedResponses/ResponsesTable/ResponsesTable.tsx

```diff
@@ -34,12 +40,80 @@ const RESPONSE_TABLE_COLUMNS: Column<ResponseColumnData>[] = [
     Header: 'Timestamp',
     accessor: 'submissionTime',
     minWidth: 200,
-    width: 300,
+    width: 250,
     disableResizing: true,
   },
 ]
+const PAYMENT_COLUMNS: Column<ResponseColumnData>[] = [
+  {
+    Header: 'Email',
+    accessor: ({ payments }) => {
+      if (!payments?.email) {
+        return ''
+      }
+      return payments.email
+    },
+    minWidth: 50,
+    width: 150,
+  },
+
+  {
+    Header: 'Paid Amount', //  (amt responder paid)
+    accessor: ({ payments }) => {
+      if (!payments) {
+        return ''
+      }
+      return `S$${centsToDollars(payments.paymentAmt)}`
+    },
+    minWidth: 50,
+    width: 75,
+  },
+
+  {
+    Header: 'Net Amount', //  (amt they receive in bank)
+    accessor: ({ payments }) => getNetAmount(payments),
+    minWidth: 50,
+    width: 75,
+  },
+
+  {
+    Header: 'Fees', //  (paid - net)
+    accessor: ({ payments }) => {
+      if (!payments?.transactionFee) {
+        return ''
+      }
+      if (payments.transactionFee < 0) {
+        return ''
+      }
+
+      return `S$${centsToDollars(payments.transactionFee)}`
+    },
+    minWidth: 50,
+    width: 75,
+  },
+  {
+    Header: 'Payout Date',
+    accessor: ({ payments }) => {
+      if (!payments) {
```

Comment:
/nits this can be shorthanded to `return !payments ? 'Pending' : payments.payoutDate`

Reply:
Hm I actually intended these to be written in the style of guard clauses to make it readable + extensible, wdyt?

Reply:
heh, both works, hence nits! LGTM pls feel free to merge